server/prometheus.go

```diff
@@ -129,8 +159,52 @@ func (pp *PrometheusProvider) execute(ctx *gin.Context) {
 			ctx.JSON(http.StatusBadRequest, warnings)
 			return
 		}
```

Comment:
Please avoid printing directly in stdout (`fmt.Println`) from your code. If there is a need to print something please use proper logs.

Reply:
ok.